packages/integration/lib/src/views/post/post_detail_screen.dart

```diff
@@ -670,7 +672,7 @@ class _LMFeedPostDetailScreenState extends State<LMFeedPostDetailScreen> {
       },
       menu: LMFeedMenu(
         menuItems: _postDetailScreenHandler!.postData?.menuItems ?? [],
-        removeItemIds: {postReportId, postEditId},
+        removeItemIds: {postReportId},
```

Comment:
this will effect default master sdk sample app, please confirm once that we have started supporting edit post in master sdk sample app

Reply:
We have added edit post screen in the last release